plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -1289,6 +1289,7 @@ fu_logitech_bulkcontroller_device_init(FuLogitechBulkcontrollerDevice *self)
 	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_TRIPLET);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
+	fu_usb_device_set_claim_retry_count(FU_USB_DEVICE(self), 100);
```

Comment:
I assume the other devices should keep the default of 5. Let me know where I should put that; there were a few spots that would have been appropriate I think.

Reply:
> I assume the other devices should keep the default of 5

I think we only added the claim retry for the bulk-controller :)

Reply:
SGTM. Silly question, where does it get initialized to 0? Some gobject magic?

Reply:
Not silly at all; all the FuUsbDevicePrivate struct gets initialized to zero when the object is created.